packages/openapi-react-query/README.md

```diff
@@ -42,25 +42,25 @@ const fetchClient = createFetchClient<paths>({
 const $api = createClient(fetchClient);
 
 const MyComponent = () => {
-  const { data, error, isLoading } = $api.useQuery(
+  const { data, error, isPending } = $api.useQuery(
```

Comment:
```suggestion
  const { data, error, isLoading } = $api.useQuery(
```

? I’m confused; this should be `isLoading`, shouldn’t it? Why the change?

Reply:
https://tanstack.com/query/latest/docs/framework/react/guides/migrating-to-v5#status-loading-has-been-changed-to-status-pending-and-isloading-has-been-changed-to-ispending-and-isinitialloading-has-now-been-renamed-to-isloading

Reply:
You have v5 as a peer dependency.

Reply:
Ah thanks for explaining! I missed that. I think we have some other docs out-of-date, then. This is good for now!